client/admin/components/users/index.vue

```diff
@@ -120,10 +135,28 @@ export default {
         action: () => api.remove(user),
         dialog: true
       });
+    },
+    selectMultiple() {
+      if (this.multiSelect && this.selectedUsers.length >= 2) {
```

Comment:
Before I start with _serious_ review toggle those conditions to avoid unnecessary indent...